Tests/Unit/Assets/getInlineLazyloadScript.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace RocketLazyload\Tests\Unit\Assets;
+
+use Brain\Monkey\Functions;
+use RocketLazyload\Assets;
+use RocketLazyload\Tests\Unit\TestCase;
+
+/**
+ * @covers RocketLazyload\Assets::getInlineLazyloadScript
+ */
+class Test_GetInlineLazyloadScript extends TestCase {
+	private $assets;
+
+	public function setUp() {
+		parent::setUp();
+		$this->assets = new Assets();
+	}
+
+	public function testShouldReturnInlineLazyloadScriptWithOptions() {
+		Functions\when( 'esc_attr' )->returnArg();
+		Functions\when( 'wp_parse_args' )->alias( function( $args, $defaults ) {
+			if ( is_object( $args ) ) {
+				$r = get_object_vars( $args );
+			} elseif ( is_array( $args ) ) {
+				$r =& $args;
+			} else {
+				parse_str( $args, $r );
+			}
+
+			if ( is_array( $defaults ) ) {
+				return array_merge( $defaults, $r );
+			}
+
+			return $r;
```

Comment:
Test should very little computation in it. Why? We know the state and conditions of what we are testing. Therefore, we know what _should_ be received as args into this function and what should be returned from it. Right?

So instead of running code to build what gets returned, return what you expect it to be.